admin/src/templates/message_test.html

```diff
@@ -0,0 +1,19 @@
+<form action="" method="POST" class="col-xs-8 configuration-form">
+
+  <div class="form-group required" ng-class="{'has-error': errors.message}">
+    <label for="message" translate><translate>Message Body</label>
```

Comment:
This doesn't look valid. What is the `<translate>` element, why do we need that and the translate attribute, and why does it not get closed?

Also, for new translations are we not using dot notation?

Reply:
Yeah. Um... autocomplete fail I guess?

Reply:
Fixed.

Reply:
This isn't a new translation it's reusing an old one.